modules/common-fs2/src/test/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/blackbox/BlackBoxTesting.scala

```diff
@@ -53,8 +53,8 @@ import com.snowplowanalytics.snowplow.enrich.common.fs2.test.TestEnvironment
 
 object BlackBoxTesting extends Specification with CatsIO {
 
-  val igluClient: Client[IO, Json] =
-    Client[IO, Json](Resolver(List(Registry.EmbeddedRegistry), None), CirceValidator)
+  def igluClient: IgluCirceClient[IO] =
+    IgluCirceClient.fromResolver[IO](Resolver(List(Registry.EmbeddedRegistry), None), cacheSize = 0).unsafeRunSync()
```

Comment:
Did you consider making this a `IO[IgluCirceClient[IO]]` and removing `unsafeRunSync()`?  I haven't looked at how the tests are implemented, so I don't know if that's reasonable or too disruptive to rewrite all the tests.

If it's very disruptive I'm OK to merge it like this.